module/documents/actor/group.mjs

```diff
@@ -0,0 +1,93 @@
+
+/**
+ * A data model and API layer which handles the schema and functionality of "group" type Actors in the dnd5e system.
+ *
+ * @example Create a new Group
+ * const g = new dnd5e.documents.Actor5e({
+ *  type: "group",
+ *  name: "Test Group",
+ *  system: {
+ *    members: ["3f3hoYFWUgDqBP4U"]
+ *  }
+ * });
+ */
+export default class GroupActor extends foundry.abstract.DataModel {
+
+  /** @override */
+  static defineSchema() {
+    const fields = foundry.data.fields;
+    return {
+      description: new fields.SchemaField({
+        full: new fields.StringField(),
+        summary: new fields.StringField()
```

Comment:
Should these be `HTMLField`. I know the behavior isn't any different, but it would communicate a bit better the type of string they contain.